cmd/pebble/cmd_run.go

```diff
@@ -39,10 +45,11 @@ The run command starts pebble and runs the configured environment.
 type cmdRun struct {
 	clientMixin
 
-	CreateDirs bool   `long:"create-dirs"`
-	Hold       bool   `long:"hold"`
-	HTTP       string `long:"http"`
-	Verbose    bool   `short:"v" long:"verbose"`
+	CreateDirs    bool   `long:"create-dirs"`
+	Hold          bool   `long:"hold"`
+	HTTP          string `long:"http"`
+	SigtermAction string `long:"sigterm"`
```

Comment:
Consider changing to `--shutdown-signal=SIGFOO` as per @cjdcordeiro's comment. Can discuss what you think of that at standup.

Reply:
I think I agree. But I am concerned `--shutdown-signal=SIGFOO` reads different, in the docker case, --stop-signal is the signal which docker sends to the container to stop it. In this case, pebble does not send SIGFOO when it shuts down but rather listens for SIGFOO to trigger the shutdown.

I am concerned also that the signals could then used for multiple actions. For example, in future we may add `--restart-signal=SIGFOO`, when both restart-signal and shutdown-signal use SIGFOO, what is the outcome of that?
`--sigterm=shutdown`, `--sigint=restart`, is less ambiguous/less prone to undefined behaviour.

So now after writing this, I think I've convinced myself that perhaps the way it is now is the better way.

Reply:
I've already done the work to move it to the docker method. See updates. What is the decision @benhoyt @hpidcock ?